luigi/scheduler.py

```diff
@@ -822,7 +822,7 @@ def add_task(self, task_id=None, status=PENDING, runnable=True,
             task.family = family
         if not getattr(task, 'module', None):
             task.module = module
-        if not task.param_visibilities:
+        if not getattr(task, 'param_visibilities', None):
```

Comment:
`hasattr(task, 'param_visibilities')` is more concise with the same effect, but either one works.

Reply:
Good point. Stick with `getattr`. Thanks!